modules/manage/partials/authentication.adoc

```diff
@@ -1590,7 +1593,12 @@ See <<Enable OIDC>> to configure the required OIDC cluster configuration propert
 
 NOTE: If you enable OIDC authentication for the Admin API, you must also <<create-superusers,create a SCRAM superuser>> so that you can use `rpk` to create ACLs. `rpk` supports only basic authentication for the Admin API. See <<Authentication for the HTTP APIs>>.
 
-To enable OIDC for the HTTP API listeners as well as basic authentication, include OIDC in the `http_authentication` cluster property list:
+You can enable OIDC for HTTP APIs in two ways:
+
+* **Cluster-wide**: Use the `http_authentication` cluster property to enable OIDC for all HTTP API listeners
+* **Per-listener**: Use the `authentication_method` property to enable OIDC for specific API listeners
+
+To enable OIDC cluster-wide for all HTTP API listeners, include both `BASIC` and `OIDC` in the `http_authentication` cluster property list:
```

Comment:
Do I actually need to set both options if I just want OIDC? Also this was the issue described in DOC-401 no? This mentions `BASIC` and `OIDC` whereas `BASIC` should probably be `http_basic` and `OIDC` => `oidc`? (unsure if it's case sensitive, but I'm looking at the below example).

Maybe I'm completely on the wrong and the different options (cluster-wide vs per-listener) have different values? In that case I'd be surprised :D

Reply:
Ah this is very confusing, why is `authentication_method: http_basic` in the node config required / what does it do? 

The `http_basic` there is then **not** the same as the basic in `rpk cluster config set http_authentication '["BASIC","OIDC"]` I guess. (This is probably the source of confusion for me).

Reply:
> Ah this is very confusing, why is authentication_method: http_basic in the node config required / what does it do?

It enables authentication for the listener.

> The http_basic there is then not the same as the basic in rpk cluster config set http_authentication '["BASIC","OIDC"] I guess. (This is probably the source of confusion for me).

Yes, it's confusing, which I pointed out at the time.

I also suggested a more informative name `http_authentication_mechanisms`, but that was considered too long.

Reply:
Hoping the note I added clarifies this for users.